packages/ocs/block-pool/BlockPoolListPage.tsx

```diff
@@ -252,8 +252,9 @@ const RowRenderer: React.FC<RowProps<StoragePoolKind, CustomData>> = ({
   const formatedDateTime = lastChecked
     ? twelveHoursdateTimeNoYear.format(new Date(lastChecked))
     : '-';
+  const compressionMode = obj.spec?.compressionMode;
   const compressionStatus: boolean =
```

Comment:
```suggestion
  const isCompressionEnabled: boolean =
```
Lets rename the variable.